### PR TITLE
L3 ht

### PR DESCRIPTION
**Pull request by taras-petushkov**
see commit messages for more details

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+});
+
+gulp.task('css', function () {
+    return gulp.src(path.css)
+        .pipe(concat('styles.css'))
+        .pipe(less())
+        .pipe(sourcemaps.init())
+        .pipe(autoprefixer())
+        .pipe(gulpif(isProd, cssnano()))
+        .pipe(sourcemaps.write(cssMapDir))
```
Comment (puzankov) [inline gulpfile.js]: Delete sourcemap from prod

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    gulp.watch(path.html, ['copy-html']);
+    gulp.watch(path.images, ['copy-images']);
+});
+
+gulp.task('style', function (cb) {
+    runSequence('style-jscs', 'style-csscomb', 'style-jshint', 'style-htmlhint', cb);
+});
+
+function gitmodified() {
+    return gitmodified0(['added', 'modified', 'untracked', 'renamed', 'copied', 'updated but unmerged']);
```
Comment (puzankov) [inline gulpfile.js]: what is 0

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+gulp.task('style-jscs', function () {
+    return gulp.src(path.scripts)
+        .pipe(gulpif(!isAll, gitmodified()))
+        .pipe(debug({ title: 'jscs' }))
+        .pipe(jscs({ fix: true }))
+        .pipe(gulp.dest(srcDir));
+});
+
```
Comment (puzankov) [inline gulpfile.js:157]: check configs usage

File: gulpfile.js (hunk 1 above)
Comment (taras-petushkov) [inline gulpfile.js]: done. my understanding of sourcemaps was wrong

File: gulpfile.js (hunk 2 above)
Comment (taras-petushkov) [inline gulpfile.js]: renamed to gitmodifiedOrigin

File: gulpfile.js (hunk 3 above)
Comment (taras-petushkov) [inline gulpfile.js:157]: checked by modifying configurations and checking gulp messages. everything works as expected

**Comment by puzankov** [comment]
+1

**Pull request merged**